RoboFile.php

```diff
@@ -618,4 +625,34 @@ public function writeReleaseVersion($version) {
       ->to(sprintf("'version' => '%s',", $version))
       ->run();
   }
+
+  public function jiraReleaseVersion($opts = ['free' => null, 'premium' => null]) {
+    require_once './tasks/release/Jira.php';
+    if (empty($opts['free']) && empty($opts['premium'])) {
+      $this->yell('No Free or Premium version specified', 40, 'red');
+      exit(1);
+    }
+    $output = [];
+    if (!empty($opts['free'])) {
```

Comment:
So far we run all commands for premium release from premium folder. I think we should stay consistent it this. So I suggest here to use only the option `version-name` (`version` is already used by Robo to print its version) hardcode project name and add this task also to premium plugin RoboFile.php.
If we also allow automatic version detection (it is mentioned in other comment) the developer can only call `./do j:r` which might be also useful for possible future usage within Circle CI.

Reply:
We can even use a nameless argument. Premium-related code removed, thanks!